yellowbrick/text/postag.py

```diff
@@ -114,252 +131,279 @@ def __init__(
+                        counter["conjunction"] += 1
                     elif tag == "CD":
-                        self.pos_tag_counts_["digit"] += 1
+                        counter["digit"] += 1
                     # combine predeterminer and determiner
                     elif tag in ["DT" or "PDT"]:
-                        self.pos_tag_counts_["determiner"] += 1
+                        counter["determiner"] += 1
                     elif tag == "EX":
-                        self.pos_tag_counts_["existential"] += 1
+                        counter["existential"] += 1
                     elif tag == "FW":
-                        self.pos_tag_counts_["non-English"] += 1
+                        counter["non-English"] += 1
                     elif tag == "IN":
-                        self.pos_tag_counts_["preposition"] += 1
+                        counter["preposition"] += 1
                     elif tag == "POS":
-                        self.pos_tag_counts_["possessive"] += 1
+                        counter["possessive"] += 1
                     elif tag == "LS":
-                        self.pos_tag_counts_["list"] += 1
+                        counter["list"] += 1
                     elif tag == "MD":
-                        self.pos_tag_counts_["modal"] += 1
+                        counter["modal"] += 1
                     elif tag in self.punct_tags:
-                        self.pos_tag_counts_["punctuation"] += 1
+                        counter["punctuation"] += 1
                     elif tag == "TO":
-                        self.pos_tag_counts_["infinitive"] += 1
+                        counter["infinitive"] += 1
                     elif tag == "UH":
-                        self.pos_tag_counts_["interjection"] += 1
+                        counter["interjection"] += 1
                     elif tag == "SYM":
-                        self.pos_tag_counts_["symbol"] += 1
+                        counter["symbol"] += 1
                     else:
-                        self.pos_tag_counts_["other"] += 1
-
+                        counter["other"] += 1
+        
     def draw(self, **kwargs):
         """
         Called from the fit method, this method creates the canvas and
         draws the part-of-speech tag mapping as a bar chart.
-
+        
         Parameters
         ----------
         kwargs: dict
             generic keyword arguments.
-
+        
         Returns
         -------
         ax : matplotlib axes
             Axes on which the PosTagVisualizer was drawn.
         """
         colors = resolve_colors(
-            n_colors=len(self.pos_tag_counts_),
-            colormap=self.colormap,
-            colors=self.colors,
-        )
-
+                    n_colors=len(self.pos_tag_counts_), 
+                    colormap=self.colormap,
+                    colors=self.colors
+                    )
+        
+        pos_tag_sum = copy.deepcopy(self.pos_tag_counts_)
```

Comment:
Why is this deep copy required? If it's necessary, I'm all for it - but we do want to be conscious of how much memory we're using when creating tools in a machine learning context -- people might be working with very large datasets.

Reply:
Deep copy is used as pos_tag_sum may be a nested list. So basic idea was to take an extra measure in case I changed the inner values. But in this I am not using it so I will go on to save it as `copy()` and not `deepcopy()`